deploy/iso/minikube-iso/board/coreos/minikube/rootfs-overlay/init

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+mkdir /sysroot
+mount -t tmpfs -o size=90% tmpfs /sysroot
```

Comment:
Add a comment explaining how you arrived at 90% being the correct choice here.

Reply:
That’s easy, I stole it from tinycore (and boot2docker). 😀
I think it needs some space left, so using 100% won’t work. But we could probably get away with 95% since our VM is twice as big ? Can experiment with it

Reply:
That's fine. Just mention that you have copied the number from elsewhere then. I was afraid it might have some magical principle to it.

Reply:
I'll stick with 90%, until I know better. Will leave a reference to the original